awx/api/serializers.py

```diff
@@ -4741,20 +4741,20 @@ def create(self, validated_data):
         WorkflowJobNode.objects.bulk_create(nodes)
 
         # Deal with the m2m objects we have to create once the node exists
-        for obj_type, obj_through_model in node_m2m_object_types_to_through_model.items():
-            through_models = []
+        for field_name, through_model in node_m2m_object_types_to_through_model.items():
+            through_model_objects = []
             for node_identifier in node_m2m_objects.keys():
-                if obj_type in node_m2m_objects[node_identifier] and obj_type == 'credentials':
-                    for cred in node_m2m_objects[node_identifier][obj_type]:
-                        through_models.append(obj_through_model(credential=cred, workflowjobnode=node_m2m_objects[node_identifier]['node']))
-                if obj_type in node_m2m_objects[node_identifier] and obj_type == 'labels':
-                    for label in node_m2m_objects[node_identifier][obj_type]:
-                        through_models.append(obj_through_model(label=label, workflowjobnode=node_m2m_objects[node_identifier]['node']))
+                if field_name in node_m2m_objects[node_identifier] and field_name == 'credentials':
+                    for cred in node_m2m_objects[node_identifier][field_name]:
+                        through_model_objects.append(through_model(credential=cred, workflowjobnode=node_m2m_objects[node_identifier]['node']))
+                if field_name in node_m2m_objects[node_identifier] and field_name == 'labels':
+                    for label in node_m2m_objects[node_identifier][field_name]:
+                        through_model_objects.append(through_model(label=label, workflowjobnode=node_m2m_objects[node_identifier]['node']))
                 # if obj_type in node_m2m_objects[node_identifier] and obj_type == 'instance_groups':
                 #     for instance_group in node_m2m_objects[node_identifier][obj_type]:
-                #         through_models.append(obj_through_model(instancegroup=instance_group, workflowjobnode=node_m2m_objects[node_identifier]['node']))
-            if through_models:
-                obj_through_model.objects.bulk_create(through_models)
+                #         through_model_objects.append(through_model_obj(instancegroup=instance_group, workflowjobnode=node_m2m_objects[node_identifier]['node']))
```

Comment:
I think this should be `through_model(instancegroup=instance_group, ...`

Given an object, it doesn't make sense to do `obj(..)`, that's something you do with classes to make an object. Looking over the diff, I think this concern is valid, as `obj_through_model` was renamed to `through_model`. Given the new code, I'm not sure if `through_model_obj` is even defined.